src/threadAccessController.js

```diff
@@ -113,25 +138,33 @@ class ThreadAccessController extends EventEmitter{
   }
 
   async save () {
-    return {
+    const manifest = {
       address: this._db.address.toString(),
       firstModerator: this._firstModerator,
       members: this._members
     }
+    if (this._encKeyId) manifest.encKeyId = this._encKeyId
+    return manifest
   }
 
-  async grant (capability, id) {
+  async grant (capability, id, ciphertext) {
```

Comment:
I don't think it's right to call this param `ciphertext` since it's an object that contains three different properties: `ciphertext`, `nonce`, `ephemeralFrom`.

Reply:
it actually is just a ciphertext string!

Reply:
hm, where do you get the nonce and ephemeralFrom params needed to decrypt the ciphertext?